pom.xml

```diff
@@ -187,6 +198,23 @@
         <apache.commons-codec.version>1.10</apache.commons-codec.version>
         <apache.commons-io.version>2.5</apache.commons-io.version>
         <apache.commons-fileupload.version>1.4</apache.commons-fileupload.version>
+
+        <!-- ************** -->
+        <!-- Build settings -->
+        <!-- ************** -->
+
+        <!-- i18n-maven-plugin -->
+        <i18nPluginVersion>1.4.1</i18nPluginVersion>
+        <!-- maven-jar-plugin http://maven.apache.org/plugins/maven-jar-plugin/ -->
+        <mavenJarPluginVersion>3.2.0</mavenJarPluginVersion>
```

Comment:
Why overriding wrensec-parent?

Reply:
Same for plugins below this one.

Reply:
I have missed the definitions in the parent pom, removed.